deeppavlov/models/embedders/elmo_embedder.py

```diff
@@ -38,9 +38,11 @@ class ELMoEmbedder(Component):
     Parameters:
         spec: A ``ModuleSpec`` defining the Module to instantiate or a path where to load a ``ModuleSpec`` from via
             ``tenserflow_hub.load_module_spec`` by using `TensorFlow Hub <https://www.tensorflow.org/hub/overview>`__.
+        elmo_output_names: A list of output ELMo. You can use combination of ``["word_emb", "lstm_outputs1", "lstm_outputs2","elmo"]`` and you can use separately ``["default"]``. See `TensorFlow Hub <https://www.tensorflow.org/hub/overview>`__ for more information about it.
```

Comment:
I would like to see a brief description for every name or a link to those descriptions as https://www.tensorflow.org/hub/overview has nothing of the sort.

Reply:
You're right, I changed the link to  a correct url.